Doc/library/unittest.mock.rst

```diff
@@ -1006,11 +1006,11 @@ object::
       Assert the mock has been awaited with the specified calls.
       The :attr:`await_args_list` list is checked for the awaits.
 
-      If *any_order* is False (the default) then the awaits must be
+      If *any_order* is false (the default) then the awaits must be
```

Comment:
```suggestion
      If *any_order* is ``False`` (the default) then the awaits must be
```

Reply:
Same as above. `any_order=0` has the same effect as `any_order=False`.

Reply:
Yes, but here you're  saying two things: semantics and the default value. Of course, for semantics it should be false, but the default value really is `False`, isn't it? If you really want to keep the distinction (and I think it's a laudable goal), you have to separate them.

If *any_order* is false (the default is `False`) then...

Reply:
I agree with Vedran about distinguishing default from allowed inputs.  Since the default False is in the signature just 4 lines above, I don't think it needs to be repeated.  Do we usually repeat them so close to the signature?

```suggestion
      If *any_order* is false, then the awaits must be
```